fitting/draw_signals.py

```diff
@@ -0,0 +1,50 @@
+import ROOT
```

Comment:
Did you mean to include this file in the PR? If so it would be good to unhardcode `TNT` strings

Reply:
Yeah I thought it could be a useful script for others to make the same plot but I was also too lazy to make it fully generic haha. I can just remove it